lib/CMakeLists.txt

```diff
@@ -255,9 +256,10 @@ check_c_source_compiles("#include <arm_neon.h>\nint main(){ uint8_t *dest; uint8
 	                neon_compile_result)
 
 if(neon_compile_result)
-    check_c_source_compiles("#include <volk/volk_common.h>\n int main(){__VOLK_ASM volatile(\"vrev32.8 q0, q0\");}"
+    set(CMAKE_REQUIRED_INCLUDES ${PROJECT_SOURCE_DIR}/include)
+    check_c_source_compiles("#include <volk/volk_common.h>\n int main(){__VOLK_ASM __VOLK_VOLATILE(\"vrev32.8 q0, q0\");}"
                             have_neonv7_result )
-    check_c_source_compiles("#include <volk/volk_common.h>\n int main(){__VOLK_ASM volatile(\"sub v1.4s,v1.4s,v1.4s\");}"
+    check_c_source_compiles("#include <volk/volk_common.h>\n int main(){__VOLK_ASM __VOLK_VOLATILE(\"sub v1.4s,v1.4s,v1.4s\");}"
```

Comment:
Did we forgot about this `__VOLK_VOLATILE` use in the previous PR?

Reply:
This all looks reasonable to me. I really need to setup something in my Jenkins to run volk builds on a few toolchains

Reply:
I thought we run ARM CI as well. Though it might be `aarch64` only. Need to check.